### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by dfsnow**
This PR updates the README to reflect the changes from #6, #7, and #8.

### CODE REVIEW

File: config/default_jobs.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 define:
-  min_year: &min_year 2022
+  min_year: &min_year 2023
```
Comment (dfsnow) [inline config/default_jobs.yaml:2]: Sneaking these changes in. If we want to match the current sqoop behavior we actually want to grab >= 2023 data.

File: README.md [hunk 2]
```diff
@@ -58,35 +78,134 @@ as a file. All jobs should have the following format:
...
-### Predicates, filtering, and partitioning
+The command line interface also has multiple optional flags:
+
+- `--run-github-workflow/--no-run-github-workflow` - Run the [`test_dbt_models`](https://github.com/ccao-data/data-architecture/blob/master/.github/workflows/test_dbt_models.yaml)
+  workflow on batch completion?
+- `--run-glue-crawler/--no-run-glue-crawler` - Run the iasWorld Glue crawler
+  on batch completion?
+- `--upload-data/--no-upload-data` - Upload extracted data to the iasWorld S3
+  bucket?
+- `--upload-logs/--no-upload-logs` - Upload batch logs to AWS CloudWatch?
```
Comment (jeancochrane) [inline README.md:124]: **[Suggestion, non-blocking]** Can we include the default values for these flags?

File: README.md [hunk 3]
```diff
@@ -58,35 +78,134 @@ as a file. All jobs should have the following format:
...
 
+### Files not included
+
+Some necessary setup and credential files are not included in this repository
+for security reasons. Templated versions are included for instructional
+purposes. If you want to use this repository, you will need to populate the
+following:
+
+- `drivers/ojdbc8.jar` - This is the JDBC driver for our Oracle backend and
+  can be found for free on [Oracle's site](https://www.oracle.com/ca-en/database/technologies/appdev/jdbc-downloads.html).
```
Comment (jeancochrane) [inline README.md:191]: **[Question, non-blocking]** I'm guessing this is excluded less for security reasons than for dependency-management reasons, is that right? Or are there credentials in the JDBC driver that aren't populated when downloading from Oracle's site?

**Review by jeancochrane — APPROVED** [review]
Nice work!

File: README.md (hunk 2 above)
Comment (dfsnow) [reply README.md:124]: Done in 51f1a71!

File: README.md (hunk 3 above)
Comment (dfsnow) [reply README.md:191]: It's actually a licensing thing. Clarified in 51f1a71.

**Pull request merged**